.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pass
 *.env
+**/.DS_Store
```

Comment:
Can you please add an empty line at the end of file? That's usually the standard for this repo, except for `.name` files (because we read them with `cat`)